src/cfnlint/data/AdditionalSpecs/LmbdRuntimeLifecycle.json

```diff
@@ -2,71 +2,126 @@
  "dotnetcore1.0": {
   "deprecated": "2019-07-31",
   "eol": "2019-06-27",
-  "successor": "dotnet6"
+  "successor": "dotnet8"
  },
  "dotnetcore2.0": {
   "deprecated": "2019-05-30",
   "eol": "2019-04-30",
-  "successor": "dotnet6"
+  "successor": "dotnet8"
  },
  "dotnetcore2.1": {
   "deprecated": "2021-09-23",
   "eol": "2021-08-23",
-  "successor": "dotnet6"
+  "successor": "dotnet8"
  },
  "dotnetcore3.1": {
   "deprecated": "2023-02-20",
   "eol": "2023-01-20",
-  "successor": "dotnet6"
+  "successor": "dotnet8"
+ },
+ "dotnet5.0": {
+  "deprecated": "2022-05-10",
+  "eol": "2022-05-10",
+  "successor": "dotnet8"
+ },
+ "dotnet6": {
+  "deprecated": "",
```

Comment:
https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-supported

Docs now split deprecation dates into 3 columns rather than 2 and mention:
`For new regions, Lambda will not support runtimes that are set to be deprecated within the next 6 months.`
Rather not maintain more dates for dozens of runtimes and skeptical that precision's needed anyways. Think we either:
1) Just continue picking 2 columns out of the 3
2) Pick a column and simply calculate based on it (`Error` on date, `Warn` months before, `Info` years before)

@kddejong thoughts?

Reply:
I like 2 cause it would make life a lot easier but I get a little nervous about exceptions to that. So may need to have an exception process as it comes up.

Reply:
> exception process

Thinking no exceptions for option 2. Most rows currently span a couple months between all 3 columns. We'd err earlier (like `Info` full year in advance, `Warn` half year in advance) and modify the wording of non-error rule(s) accordingly:

https://github.com/aws-cloudformation/cfn-lint/blob/dba16cb09e142d89144577aa6505e8871589e2d0/src/cfnlint/rules/resources/lmbd/DeprecatedRuntimeEol.py#L36

If timeframes between columns increase beyond that, we'd increase advance notices for all runtimes. We haven't updated these dates in almost 2 years; it's more important to give devs more notice than bother translating all the specific documentation columns. Devs can read docs themselves as long as we surface some notice